src/atoms/weavemol.jl

```diff
@@ -1,13 +1,19 @@
 # some commentary
 
 # docstring...
+
+# represents a given pair of atoms by their respective indices in the adjacency matrix representation
+const PairIndexes = Tuple{Integer,Integer}
+
 mutable struct WeaveMol <: AbstractAtoms
     smiles::String
     elements::Vector{String}
-    atom_features::Vector{SomethingOrOther} # I need to look more carefully to figure this out, heh
-    pair_features::Vector{SomethingOrOther}
+    atom_features::Vector{SomethingOrOther} # Details TBD
+    # pairs encapsulates the PairFeature features for a given pair of atoms in a molecule represented using PairIndexes
+    pairs::Vector{Tuple{PairIndexes,PairFeature}}
```

Comment:
similar to above, I don't think this needs to be explicitly stored since graph iteration functions are pretty well-optimized

Reply:
Fixed in e4beb74.